mycore-pi/src/test/java/org/mycore/pi/condition/MCRPIAndPredicateTest.java

```diff
@@ -0,0 +1,70 @@
+package org.mycore.pi.condition;
+
+import java.util.Map;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.mycore.common.MCRTestCase;
+import org.mycore.datamodel.metadata.MCRBase;
+import org.mycore.pi.MCRPIJobService;
+
+import static org.junit.Assert.*;
+
+public class MCRPIAndPredicateTest extends MCRTestCase {
+
+    private static final String MOCK_SERVICE_1 = "MOCK1";
+    private static final String MOCK_SERVICE_2 = "MOCK2";
+    private static final String MOCK_SERVICE_3 = "MOCK3";
+    private static final String MOCK_SERVICE_4 = "MOCK4";
+
+    @Test
+    public void test(){
+        final MCRBase emptyBase = new MCRBase() {
+            @Override protected String getRootTagName() {
+                return "null";
+            }
+        };
+        final boolean mock1Result = MCRPIJobService
+            .getPredicateInstance("MCR.PI.Service." + MOCK_SERVICE_1 + ".CreationPredicate").test(emptyBase);
+        final boolean mock2Result = MCRPIJobService
+            .getPredicateInstance("MCR.PI.Service." + MOCK_SERVICE_2 + ".CreationPredicate").test(emptyBase);
+        final boolean mock3Result = MCRPIJobService
+            .getPredicateInstance("MCR.PI.Service." + MOCK_SERVICE_3 + ".CreationPredicate").test(emptyBase);
+        final boolean mock4Result = MCRPIJobService
+            .getPredicateInstance("MCR.PI.Service." + MOCK_SERVICE_4 + ".CreationPredicate").test(emptyBase);
+
+        Assert.assertTrue(mock1Result);
+        Assert.assertFalse(mock2Result);
+        Assert.assertTrue(mock3Result);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'Assert.assertTrue' due to existing static import 'org.junit.Assert.*'](https://app.codacy.com/manual/MyCoRe/mycore/pullRequest?prid=5472577)